spec/core/environment_spec.rb

```diff
@@ -38,7 +38,16 @@
         allow(Dir).to receive(:glob).and_return([])
         expect { subject.load_runtime_jars! }.to raise_error
       end
+    end
+  end
+
+  context "plugin paths" do
```

Comment:
In https://github.com/colinsurprenant/logstash/pull/1 I proposed a way to improve/organize a bit better the code in this test. It does not means this test are wrong, but the way of writing this I think i could be improved. Let me know what do you think?.